src/ResourceManager/Network/Commands.Network/VirtualNetworkGatewayConnection/NewAzureVirtualNetworkGatewayConnectionCommand.cs

```diff
@@ -134,7 +134,7 @@ public class NewAzureVirtualNetworkGatewayConnectionCommand : VirtualNetworkGate
         [Parameter(
             Mandatory = false,
             HelpMessage = "Use policy-based traffic selectors for a S2S connection")]
-        public SwitchParameter UsePolicyBasedTrafficSelectors { get; set; }
+        public string UsePolicyBasedTrafficSelectors { get; set; }
```

Comment:
UsePolicyBasedTrafficSelectors has not been documented and exposed to customer so this should be an okay change. We are normalizing the parameter to the same use as our other Boolean flags. See the below comments.